portlets/src/main/webapp/vue-app/rules/components/drawers/RuleContentFormDrawer.vue

```diff
@@ -46,232 +45,275 @@
+                      </v-card-text>
+                      <div v-if="canVariableRewarding" class="d-flex justify-center">
+                        <v-chip
+                          class="ma-2"
+                          :color="!variablePoints && 'primary' || ''"
+                          :outlined="variablePoints"
+                          :dark="!variablePoints"
+                          @click="selectRewardingMode">
+                          {{ $t('rule.form.label.fixedPoints') }}
+                        </v-chip>
+                        <v-chip
+                          class="ma-2"
+                          :color="variablePoints && 'primary' || ''"
+                          :outlined="!variablePoints"
+                          :dark="variablePoints"
+                          @click="selectRewardingMode">
+                          {{ $t('rule.form.label.variablePoints') }}
+                        </v-chip>
+                      </div>
+                      <div class="d-flex flex-row">
+                        <v-card
+                          flat
+                          class="d-flex flex-grow-1 pa-0 col-4">
+                          <v-text-field
+                            v-model="rule.score"
+                            :rules="scoreRules"
+                            class="mt-0 pt-0 me-2"
+                            type="number"
+                            outlined
+                            dense
+                            required />
+                        </v-card>
+                        <v-card-text class="mt-1 px-0 col-2"> {{ canVariableRewarding && variablePoints ? $t('rule.form.label.pointsFor') : $t('rule.form.label.points') }} </v-card-text>
+                        <v-card
+                          v-if="canVariableRewarding && variablePoints"
+                          flat
+                          class="d-flex flex-grow-1 pt-2 pe-0 col-4">
+                          <v-text-field
+                            v-model="totalTargetItem"
+                            class="mt-0 pt-0 me-2"
+                            type="number"
+                            outlined
+                            dense
+                            required />
+                        </v-card>
+                        <v-card-text v-if="canVariableRewarding && variablePoints" class="mt-1 px-0 col-2"> {{ targetItem }} </v-card-text>
+                      </div>
+                    </div>
+                  </v-slide-y-transition>
+                </v-stepper-items>
+              </div>
+            </div>
+            <div :class="expanded && 'pt-6'">
+              <div :class="!expanded && 'pt-6'">
+                <v-stepper-step
+                  :step="4"
+                  class="ma-0 pa-0"
+                  :editable="ruleTitleValid && firstStepValid && !expanded">
+                  <span class="font-weight-bold text-header">{{ $t('rule.form.label.stepFour') }}</span>
```

Comment:
please reuse generic stepper font styles
```suggestion
                  {{ $t('rule.form.label.stepFour') }}
```